erdjs/src/smartcontracts/codec/index.ts

```diff
@@ -4,3 +4,4 @@
  */
 
 export * from "./binary";
+export * from "./sugar";
```

Comment:
sugar? :)

Reply:
I've derived the name from "syntactic sugar" :cookie: :candy:  

In our case, it's more like "construction sugar" (sweet trivial factory functions).

Can we keep as it is :) ?

Reply:
Fixed.